packages/react/src/components/button/Button.stories.tsx

```diff
@@ -53,29 +92,89 @@ FullWidth.storyName = 'Full width';
 
 export const Icons = () => (
   <>
-    <Button onClick={onClick} iconLeft={<IconShare />}>
+    <Button onClick={onClick} iconStart={<IconShare />}>
       Button
     </Button>
 
-    <br />
-    <br />
+    <Button onClick={onClick} iconEnd={<IconAngleRight />}>
+      Button
+    </Button>
 
-    <Button onClick={onClick} iconRight={<IconAngleRight />}>
+    <Button onClick={onClick} iconStart={<IconShare />} iconEnd={<IconAngleRight />}>
       Button
     </Button>
 
-    <br />
-    <br />
+    <Button onClick={onClick} iconStart={<IconShare />} size={ButtonSize.Small}>
+      Button
+    </Button>
 
-    <Button onClick={onClick} iconLeft={<IconShare />} iconRight={<IconAngleRight />}>
+    <Button onClick={onClick} iconEnd={<IconAngleRight />} size={ButtonSize.Small}>
+      Button
+    </Button>
+
+    <Button onClick={onClick} iconStart={<IconShare />} iconEnd={<IconAngleRight />} size={ButtonSize.Small}>
       Button
     </Button>
   </>
 );
 
+export const Themes = () => (
+  <>
+    <Button onClick={onClick} theme={ButtonPresetTheme.Coat}>
+      Coat
+    </Button>
+
+    <Button onClick={onClick} theme={ButtonPresetTheme.Coat} variant={ButtonVariant.Secondary}>
+      Coat
+    </Button>
+
+    <Button
+      onClick={onClick}
+      theme={ButtonPresetTheme.Coat}
+      variant={ButtonVariant.Supplementary}
+      iconEnd={<IconAngleRight />}
+    >
+      Coat
+    </Button>
+
+    <Button theme={ButtonPresetTheme.Black} onClick={onClick} variant={ButtonVariant.Primary}>
+      Black
+    </Button>
+
+    <Button theme={ButtonPresetTheme.Black} onClick={onClick} variant={ButtonVariant.Secondary}>
+      Black
+    </Button>
+
+    <Button
+      theme={ButtonPresetTheme.Black}
+      onClick={onClick}
+      iconEnd={<IconAngleRight />}
+      variant={ButtonVariant.Supplementary}
+    >
+      Black
+    </Button>
+  </>
+);
+
+export const CustomTheme = () => {
+  const customTheme: ButtonTheme = {
+    '--background-color-active': 'darkorange',
+    '--background-color-focus': 'orange',
+    '--background-color-hover': 'orange',
+    '--background-color': 'grey',
+    '--border-color-active': 'darkorange',
+    '--border-color': 'grey',
+  };
+  return (
+    <Button onClick={onClick} theme={customTheme} variant={ButtonVariant.Primary}>
+      Custom
+    </Button>
+  );
+};
+
 export const Loading = () => (
-  <Button isLoading loadingText="Saving your changes">
-    Button
+  <Button iconStart={<LoadingSpinner small />} variant={ButtonVariant.Loading}>
+    Saving your changes
```

Comment:
All good! But... I'd consider naming the "Loading"-variant to something else like "Clear", "Plain", "Text" etc.

Reply:
Hmm... btw we could even get rid of setting the iconStart by using the variant? But then it should be loading-something? @NikoHelle ?